x-pack/plugins/security/public/account_management/account_management_app.ts

```diff
@@ -7,7 +7,6 @@
 import { i18n } from '@kbn/i18n';
 import { StartServicesAccessor, ApplicationSetup, AppMountParameters } from 'src/core/public';
 import { AuthenticationServiceSetup } from '../authentication';
```

Comment:
__note:__ decided to lazy load these API clients as well - not a big win, but doesn't cost too much either.